pkg/agent/kubermatic/v1/agent.go

```diff
@@ -225,29 +227,16 @@ func clusterFromKube(kn kubermaticv1.Cluster, seedName string) (Cluster, error)
 		return Cluster{}, err
 	}
 
-	var cniPlugin CNIPluginSettings
-	if kn.Spec.CNIPlugin != nil {
-		cniPlugin.Type = kn.Spec.CNIPlugin.Type.String()
-		cniPlugin.Version = kn.Spec.CNIPlugin.Version
-	}
-
-	var clusterNetworkingConfig ClusterNetworkingConfig
-	clusterNetwork := kn.Spec.ClusterNetwork
-	clusterNetworkingConfig.IPFamily = string(clusterNetwork.IPFamily)
-	if clusterNetwork.KonnectivityEnabled != nil {
-		clusterNetworkingConfig.KonnectivityEnabled = *clusterNetwork.KonnectivityEnabled
-	}
-
 	var opaEnabled bool
 	opaIntegration := kn.Spec.OPAIntegration
 	if opaIntegration != nil {
 		opaEnabled = opaIntegration.Enabled
 	}
 
-	var userSSHKeyAgentEnabled bool
+	var enableUserSSHKeyAgent bool
 	enableUserSSHKeyAgentPointer := kn.Spec.EnableUserSSHKeyAgent
-	if enableUserSSHKeyAgentPointer != nil {
-		userSSHKeyAgentEnabled = *enableUserSSHKeyAgentPointer
+	if kn.Spec.EnableUserSSHKeyAgent != nil {
+		enableUserSSHKeyAgent = *enableUserSSHKeyAgentPointer
```

Comment:
```suggestion
	if kn.Spec.EnableUserSSHKeyAgent != nil {
		enableUserSSHKeyAgent = *kn.Spec.EnableUserSSHKeyAgent
```

Reply:
but why not use variable, it is the same thing right?

Reply:
i mean its just shorter, but personal preference i guess

Reply:
Yes. So should i change this or not?

Reply:
if you prefer it as it was, leave it. I just gave my suggestion :)

Reply:
Okay please approve the pr if it looks good